components/patient-data/ui/src/main/resources/PhenoTips/AllergiesDataClass.xml

```diff
@@ -72,7 +72,7 @@
   #if($xcontext.action == 'edit' || $xcontext.action == 'inline')
     {{html wiki=false clean=false}}&lt;input class="xwiki-free-multiselect-value" type="text" size="30" name="${prefix}${name}" id="${prefix}${name}_${counter}" value="$!escapetool.xml($v)"/&gt;{{/html}}##
   #else
-    $escapetool.xml($v)##
+    $!{services.rendering.escape($v, $doc.syntax)}##
```

Comment:
Just curious what is the difference between `escapetool.xml` and `services.rendering.escape` and where should we use one  and when the other? E.g. in the previous file `escapetool.xml` was added, but here it is replaced